### PR TITLE
Follow up #38

### PR DESCRIPTION
**Pull request by recena**
@reviewbybees esp. @jglick since he noticed of this [bug](https://github.com/jenkinsci/branch-api-plugin/pull/38#discussion_r61626959)

### CODE REVIEW

File: src/main/java/jenkins/branch/OrganizationFolder.java [hunk 1]
```diff
@@ -263,9 +263,20 @@ public View getPrimaryView() {
         return super.getPrimaryView();
     }
 
+    /**
+     * Creates a place holder view when there's no active branch indexed.
+     */
+    protected View getWelcomeView() {
+        return new MultiBranchProjectEmptyView(this);
```
Comment (jglick) [inline src/main/java/jenkins/branch/OrganizationFolder.java]: :bug: I think you meant `new OrganizationFolderEmptyView(this)`, which `getPrimaryView` should be made to use.

Comment (recena) [inline src/main/java/jenkins/branch/OrganizationFolder.java]: @jglick I'm sorry, I lost my concentration some days ago.

**Comment by recena** [comment]
@jglick Do you agree if we move this logic about EmptyViews to `ComputedFolder`?

File: src/main/java/jenkins/branch/OrganizationFolder.java [hunk 2]
```diff
@@ -263,9 +263,20 @@ public View getPrimaryView() {
         return super.getPrimaryView();
     }
 
+    /**
```
Comment (jglick) [inline src/main/java/jenkins/branch/OrganizationFolder.java]: :ant: ought to reuse this from `getPrimaryView()`.

**Comment by jglick** [comment]
:bee:; assuming it works.

**Comment by jglick** [comment]
> Do you agree if we move this logic about EmptyViews to `ComputedFolder`?

Seems like a good idea. Not in this PR of course, but later.

**Comment by amuniz** [comment]
:bee:

File: src/main/java/jenkins/branch/OrganizationFolder.java (hunk 2 above)
Comment (recena) [inline src/main/java/jenkins/branch/OrganizationFolder.java]: @jglick Done https://github.com/jenkinsci/branch-api-plugin/pull/39/commits/db00b5f63da3201ac427110822ff9b23dc95cc96 I tried to avoid unnecessary changes but obviously, it makes sense the change.

**Comment by recena** [comment]
@amuniz, @jglick a re-bee from you guys, would be nice.

**Comment by recena** [comment]
@jglick FYI https://issues.jenkins-ci.org/browse/JENKINS-34597 I don't know if the description is clear. Please, feel free to add any comment.

**Comment by jglick** [comment]
re:bee:

**Comment by andresrc** [comment]
:bee:

**Comment by recena** [comment]
@reviewbybees done
